pkg/crc/constants/constants_darwin.go

```diff
@@ -23,5 +23,5 @@ func TrayExecutablePath() string {
 		return filepath.Clean(filepath.Join(version.InstallPath(), "..", "MacOS", "crc-tray"))
 	}
 	// Should not be reached, tray is only supported on installer builds
-	return filepath.Clean(filepath.Join(BinDir(), "CodeReady Containers"))
+	return filepath.Clean(filepath.Join(BinDir(), "CRC"))
```

Comment:
@anjannath This should also change as per tray side changes..

Reply:
The name tray remains `crc-tray.exe`, but internally named "Red Hat OpenShift Local"

Reply:
@gbraad This is not the file name but folder name.

Reply:
yep... changed